docker/Dockerfile.local

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 Tigris Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:jammy-20220531 AS build
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gcc \
+    git \
+    golang \
+    make \
+    sudo
+
+RUN mkdir /build
+
+#Download deps once, during docker build. Rebuild only on go.mod change
+ENV PATH="${PATH}:/root/go/bin"
+COPY scripts/install_build_deps.sh /build
+RUN sh /build/install_build_deps.sh
+COPY go.mod /build
+WORKDIR /build
+RUN go mod download
+COPY . /build
+RUN --mount=type=cache,target=/root/.cache/go-build rm -f server/service && make bins
+
+
+FROM ubuntu:jammy-20220531 AS server
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+	curl \
+	tree
+
+ADD https://dl.typesense.org/releases/0.23.0/typesense-server-0.23.0-amd64.deb /tmp
```

Comment:
We would need to install platform specific packages here and for FDB below?

Reply:
Right.